modules/data-entry/src/main/frontend/src/questionnaire/Section.jsx

```diff
@@ -101,6 +101,18 @@ function Section(props) {
     sectionDefinition,
     formContext);
 
+  // Determine if the section has any answers
+  let hasAnswers = isEdit;
+  if (!isEdit && existingAnswer[0]) {
+    Object.entries(existingAnswer[0][1]).forEach( ([key, item]) => {
+      if (item.displayedValue) {
```

Comment:
Same comment about answers just with notes.

```suggestion
      if (item.displayedValue || item.note) {
```